src/NovelRT/Experimental/EngineConfig.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#include <NovelRT/Experimental/EngineConfig.h>
+
+namespace NovelRT
```

Comment:
This is available from the root namespace?

Reply:
I had my reasons for this but don't remember them at present.

Reply:
Looks like for future use outside of ECS (plugins?). Not a priority atm, just a question